egs/tedlium/s5_r2_wsj/local/chain/tuning/run_tdnn_lstm_1a.sh

```diff
@@ -0,0 +1,270 @@
+#!/bin/bash
+
```

Comment:
@danpovey Should I remove all these nnet3 files copied over from s5_r2, since s5_r2_wsj is only supposed to show training Tedlium using WSJ GMM models?